geronimo-openapi-impl/src/test/java/org/apache/geronimo/microprofile/openapi/impl/processor/SchemaProcessorTest.java

```diff
@@ -16,25 +16,24 @@
  */
 package org.apache.geronimo.microprofile.openapi.impl.processor;
 
-import static java.util.Arrays.asList;
```

Comment:
Maybe adjust import order to avoid the diff?

Reply:
DONE